docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Use Codewind in CodeReady Workspaces to build high quality cloud native applications."
+permalink: codewind-crw-quick-guide.html
+duration: 10 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
+## Objectives
+* Install CodeReady Workspaces and Codewind
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local machine.  
+
+### Prerequisite
+CodeReady Workspaces require at least two 5Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 5Gi RWO volume for each created workspace.
+
+Each Codewind workspace also requires at least on 5Gi ReadWriteMany (RWX) persistent volume.
+
+Before you can use Codewind with CodeReady Workspaces, apply the Codewind cluster role with `oc apply -f`. 
+
+### Installing CodeReady Workspaces
+1\. Subscribe to the CodeReady Workspaces OperatorHub then install CodeReady Workspaces:
```

Comment:
@johnmcollier, @naveenkaratekid was going through the guide, and we were both stuck at this first step. How do we subscribe to CRW OperatorHub then install CRW? From an online search, I did not find any CRW OperatorHub.

Reply:
I also wasn't able to apply the yaml file in step 2. I get this error: 
```
Naveens-MacBook-Pro:codeready naveenk@ibm.com$ oc apply -f file.yaml
error: unable to recognize "file.yaml": no matches for kind "CheCluster" in version "org.eclipse.che/v1"
```

Reply:
The steps here aren't quite accurate:
1. Log on to your OpenShift cluster
2. From the sidebar, click on `Operators` then `OperatorHub`
3. Search for `CodeReady Workspaces`
4. Select it and click install

Reply:
I'll get these new instructions in shortly.